packages/telemetry/src/base/config-state.ts

```diff
@@ -6,7 +6,7 @@ import packageJson from '../../package.json';
  */
 export const TelemetrySettings = {
     // Target Azure apps insight destination
-    azureInstrumentationKey: '',
+    azureInstrumentationKey: process.env.AZURE_PROD_KEY ?? '',
```

Comment:
Not sure whether this is required? @devinea

Reply:
I think this could be useful.

Reply:
I removed this since I've reverted the change to remove resourceId from `ToolsSuiteTelemetryInitSettings`. It could be added via env var like in the example/index.ts